infrastructure/scripts/create-testdata/create-release.sh

```diff
@@ -102,6 +105,8 @@ curl http://localhost:${FRONTEND_PORT}/release \
   --form-string "application=$name" \
   --form-string "source_commit_id=${commit_id}" \
   --form-string "source_author=${author}" \
+  --form-string "previous_commit_id=${prev}" \
+  --form-string "next_commit_id=${next}" \
```

Comment:
the next commit most likely does not exist yet when `create-release.sh` is called.
```suggestion
```